dask_cuda/disk_io.py

```diff
@@ -169,12 +169,16 @@ def disk_write(path: str, frames: Iterable, shared_filesystem: bool, gds=False)
         import kvikio
 
         with kvikio.CuFile(path, "w") as f:
+            futs = []
             for frame, length in zip(frames, frame_lengths):
-                f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0).get()
+                futs.append(
+                    f.pwrite(buf=frame, count=length, file_offset=0, buf_offset=0)
+                )
+            for each_fut in futs:
+                each_fut.get()
```

Comment:
This could be replaced be replaced by a collective wait once available ( https://github.com/rapidsai/kvikio/issues/79 )